yt_dlp/extractor/naver.py

```diff
@@ -140,10 +144,11 @@ class NaverIE(NaverBaseIE):
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        content = self._download_json(
-            'https://tv.naver.com/api/json/v/' + video_id,
-            video_id, headers=self.geo_verification_headers())
-        player_info_json = content.get('playerInfoJson') or {}
+        content = self._download_webpage(f'https://tv.naver.com/api/json/v/{video_id}', video_id,
+                                         headers=self.geo_verification_headers())
+        player_info_json = self._parse_json(
+            self._search_regex(r'\"playerInfoJson\"\s+\:\s+(\{.*\})', content, 'Player Info Json'), video_id,
+            transform_source=js_to_json)
```

Comment:
It is not javascript! The issue with #3174 is in the core code

Reply:
@pukkandan Actually, the bug raise because the field wrapInfo and cateList, but we dont need that value of these fields, we only need get playerInfoJson to extract info of website.

![image](https://user-images.githubusercontent.com/36532499/159825170-240d9cdd-e8da-4322-a410-a640d68448fb.png)

Reply:
but why make a workaround when we can fix the actual issue?

Reply:
700ccbe3f14e9b1f3708a562efc57361c14dee68

Is there any part of this PR still worth merging?

Reply:
@pukkandan 👍🏻 sure.